badge/badge.css

```diff
@@ -0,0 +1,48 @@
+/* Badge CSS Start Here */
+
+.badge {
+  position: relative;
+  display: inline-block;
+}
+
+.badge-counter {
+  align-items: center;
+  display: flex;
+  font-size: 1rem;
+  font-weight: bold;
+  justify-content: center;
+  height: 1rem;
+  width: 1rem;
+  padding: 0.2rem;
+  position: absolute;
+  border-radius: 50%;
+}
+
+.badge-top-right {
+  top: 1%;
+  right: 1%;
+}
+
+.badge-sm {
+  font-size: 0.8rem;
+}
+
+.badge-md {
+  font-size: 1rem;
+}
+
+.badge-lg {
+  font-size: 1.6rem;
+}
+
+.badge .badge-red {
+  color: white;
+  background-color: var(--danger-color);
+}
+
+.badge-green {
+  color: white;
+  background-color: green;
+}
+
```

Comment:
Why do you need to add both .badge .badge-red to apply the styles of danger? It can be done as the below class by only adding .badge-green

Reply:
yes, thanks i will update this. Thanks